packages/kolibri-tools/lib/cli.js

```diff
@@ -385,15 +385,11 @@ program
             return Promise.all(
               matches.map(globbedFile => {
                 if (!patternCheck || patternCheck.match(globbedFile)) {
-                  return runLinting(globbedFile)
-                    .then(formatted => {
-                      return formatted.code;
-                    })
```

Comment:
Why was this `then` block causing the issue?

Reply:
Because it was taking the resolved value of the promise which is no longer an object with a `code` attribute, but now just the code itself, and then trying to read a property from it.